docs: add clarification to ruleset matcher (#223)
go-vela/docs: content/reference/yaml/steps.md

      continue: true
```

```yaml
---
steps:
  - ruleset:

>       # Below is displaying it will overwrite Vela's default behavior to use a 

Reviewer: `# Below is displaying it will overwrite Vela's default behavior to use a` is odd to read, IMO (i know it's not part of your changeset but it stuck out in this PR). How about `# As shown below this will overwrite Vela's default behavior to use a`?

Author: I think this is a good suggestion, let's get it in before we forget about it.

Author: I'm going to make the other comments consistent with this as well

      operator: or
```

```yaml
---
steps:
  - ruleset:

>       # Below is displaying it will is an explicit way to tell the ruleset

Reviewer: ```suggestion
      # As shown below, this is an explicit way to tell the ruleset
```
Similar to my other comment. Thoughts?

Author: I think this is a good suggestion as well, let's get it in before we forget about it.
